spec/classes/init_spec.rb

```diff
@@ -1546,5 +1566,52 @@
     end
 
   end
+  describe 'variable type and content validations' do
+    # set needed custom facts and variables
+    let :facts do
+      {
+        :kernel            => 'Linux',
+        :osfamily          => 'RedHat',
+        :lsbmajdistrelease => '6',
```

Comment:
@sharlander sorry, this code was written before I worked on the lsb deprecation... change it to ```operatingsystemrelease``` and you should be done.